README.md

```diff
@@ -63,16 +63,41 @@ const thx = msgFV["no"];
 
 **Cluster**
 
+> This is an experimental feature and may be subject to breaking changes even after a
```

Comment:
nitpick: APIs such as `clusterMeet` and `clusterNodes` are not treated as experimental features. Could you remove this part?
```suggestion
```